logviewer/src/main/resources/application.properties

```diff
@@ -0,0 +1,10 @@
+server.port=${PORT:5050}
+logging.level.org.atmosphere=warn
+logging.level.root=info
+logging.file.path=./logviewer/logs
+spring.mustache.check-template-location=false
+
+# To improve the performance during development.
+# For more information https://vaadin.com/docs/flow/spring/tutorial-spring-configuration.html#special-configuration-parameters
+# vaadin.whitelisted-packages= org/vaadin/example
+
```

Comment:
As a note to this config file, in the main application module we decided (per poll in GH Issues) to use JSON config files that are deserialized into POJOs using Jackson.

Reply:
Those are the properties for Spring, that are not supposed to be configurable for an user of the viewer

Reply:
think log4j2.xml

Reply:
They are configurable for the guy setting up the application though. That is what I mean and that is also what I was referring to. If its not possible to change that easily, we have to stick with it. But if it is, we should align it.

Reply:
While it is possible to convert some of these to an json-based Property File, I think that would be a bad choice for these.
These are not supposed to be changed even by someone setting up the application (Except the logging stuff).

I.e. changing the Port here, would result that you need to also change it in the logging-xml of the bot, in the Discord-OAuth config and in the config.json were the configurable Properties are living in.

The mustache property disables an error/warning message from being displayed. 

Converting all (future) Properties to something Bean-Based is not something that makes sense.

All Properties that are supposed to be configurable for someone that is setting up the Viewer are already in the `config.json`